nipoppy/workflows/dataset_init.py

```diff
@@ -154,20 +155,37 @@ def _init_manifest_from_bids_dataset(self) -> None:
                     if x.is_dir() and x.name.startswith(BIDS_SESSION_PREFIX)
                 ]
             )
-            if not session_ids:
-                self.logger.warning(
-                    f"Skipping subject '{ppt}': could not find a session level folder."
-                )
-                continue
+            if len(session_ids) == 0:
+                # if there are no session folders
+                # we will add a fake session for this participant
+                session_ids = [FAKE_SESSION]
```

Comment:
It would be good to write a warning to the log, something like `f"Could not find session-level folder(s) for participant {ppt}, using {FAKE_SESSION} in the manifest`"